datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1072,6 +1125,64 @@ input DatasetUpdateInput {
     Update to institutional memory, ie documentation
     """
     institutionalMemory: InstitutionalMemoryUpdate
+
+    """
+    Update to global tags
+    """
+    globalTags: GlobalTagsUpdate
+
+    """
+    Update to editable schema metadata of the dataset
+    """
+    editableSchemaMetadata: EditableSchemaMetadataUpdate
+}
+
+input EditableSchemaMetadataUpdate {
+    editableSchemaFieldInfo: [EditableSchemaFieldInfoUpdate!]!
+}
+
+input EditableSchemaFieldInfoUpdate {
+    """
+    Flattened name of a field identifying the field the editable info is applied to
+    """
+    fieldPath: String
```

Comment:
Is this required?

Reply:
If not provided will we be able to correctly associate with a field in the non-editable schema?